store/postgres/src/chain_head_listener.rs

```diff
@@ -56,6 +68,10 @@ impl ChainHeadUpdateListener {
                                 notification.payload
                             )
                         });
+                    metrics.set_chain_head_number(
```

Comment:
This deserves a comment IMHO

Reply:
@jannis, I've added a comment describing the action and intent of using set_chain_head_number() here, please confirm the content is clear and then we should be good to merge this.

Reply:
Yup, that works!